lib/order.rb

```diff
@@ -0,0 +1,37 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :basket, :basket_total
```

Comment:
I'm looking through your code and can't find anything that uses :basket_total - is it something you're planning on using, or something that got deleted?